scripts/o3de/o3de/manifest.py

```diff
@@ -22,13 +22,17 @@
 logging.basicConfig(format=utils.LOG_FORMAT)
 
 # Directory methods
+override_home_folder = None
```

Comment:
The override home folder stuff was removed

Reply:
I agree with @byrcolin, we don't need the ability to override the home folder.

Reply:
OK I wasn't sure if it was there for some kind of unit test or something